src/Components/Quiz.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+
+const Quiz = ({ questions }) => {
+  const [currentQuestion, setCurrentQuestion] = useState({});
+  const [questionsControl, setQuestionsControl] = useState([]);
+  const [answer, setAnswer] = useState(null);
+
+  const setNewQuestion = (questions) => {
+    if (questions && questions.length > 0) {
+      const questionIndex = Math.floor(Math.random() * questions.length);
+      setCurrentQuestion(questions[questionIndex]);
+
+      const tempQuestions = [...questions];
+      tempQuestions.splice(questionIndex, 1);
+      setQuestionsControl(tempQuestions);
+      setAnswer(null);
+    }
+  };
+
+  useEffect(() => {
+    setNewQuestion(questions);
+  }, [questions]);
+
+  const handleAnswer = (e) => {
+    const userAnswer = e.target.innerText;
+    setAnswer(userAnswer.toUpperCase() === currentQuestion.answer.toUpperCase());
```

Comment:
ooo smart thinking, with the .toUpperCase

Reply:
I changed to uppercase first on my scss file then my quiz broke. It took me a while to figure out what I did wrong 😮‍💨 God bless the console.log lol